### PR TITLE
Enable search by internal reference

### PR DESCRIPTION
**Pull request by rubenarakelyan**
JIRA ticket: https://regulatorydelivery.atlassian.net/browse/COSBETA-1806

## Description

Adds the ability to search product notifications by internal reference.

## Review apps

https://cosmetics-pr-2809-submit-web.london.cloudapps.digital/
https://cosmetics-pr-2809-search-web.london.cloudapps.digital/

### CODE REVIEW

File: cosmetics-web/app/models/notification.rb [hunk 1]
```diff
@@ -126,7 +126,7 @@ def self.completed
 
   def as_indexed_json(*)
     as_json(
-      only: %i[product_name notification_complete_at reference_number],
+      only: %i[product_name notification_complete_at reference_number industry_reference],
```
Comment (scruti) [inline cosmetics-web/app/models/notification.rb:129]: Maybe this is a dumb question (I don't have much context in this area), but shouldn't we aff `industry_reference` to the `mapping` block too?

Comment (slawosz) [reply cosmetics-web/app/models/notification.rb:129]: We do.

**Review by slawosz — APPROVED** [review]

**Review by nasirkhanbeis — APPROVED** [review]
Tested this on the review app, looking good as expected.

**Comment by nasirkhanbeis** [comment]
wonder if we want o search by EU reference too?

**Comment by rubenarakelyan** [comment]
> wonder if we want o search by EU reference too?

Will ask in Slack

**Pull request merged**